e2e/tests/functional/plugins/telemetryTable/telemetryTable.e2e.spec.js

```diff
@@ -45,6 +45,86 @@ test.describe('Telemetry Table', () => {
     await expect(rows).toHaveCount(50);
   });
 
+  test('on load, auto scrolls to top for descending, and to bottom for ascending', async ({
+    page
+  }) => {
+    async function getScrollPosition(top = true) {
+      const tableBody = page.locator('.c-table__body-w');
+
+      // Wait for the scrollbar to appear
+      await tableBody.evaluate((node) => {
+        return new Promise((resolve) => {
+          function checkScroll() {
+            if (node.scrollHeight > node.clientHeight) {
+              resolve();
+            } else {
+              setTimeout(checkScroll, 100);
+            }
+          }
+          checkScroll();
+        });
+      });
+
+      // make sure there are rows
+      const rows = page.getByLabel('table content').getByLabel('Table Row');
+      await rows.first().waitFor();
+
+      // wait a bit to give time for new rows to be added
+      await page.waitForTimeout(1000);
+
+      const { scrollTop, clientHeight, scrollHeight } = await tableBody.evaluate((node) => ({
+        scrollTop: node.scrollTop,
+        clientHeight: node.clientHeight,
+        scrollHeight: node.scrollHeight
+      }));
+
+      // eslint-disable-next-line playwright/no-conditional-in-test
+      if (top) {
+        return scrollTop;
+      } else {
+        return Math.abs(scrollHeight - (scrollTop + clientHeight));
+      }
+    }
+
+    const sineWaveGenerator = await createDomainObjectWithDefaults(page, {
+      type: 'Sine Wave Generator',
+      parent: table.uuid
+    });
+
+    // verify in telemetry table object view
+    await page.goto(table.url);
+    await setTimeConductorMode(page, false);
```

Comment:
if we're not testing the time conductor specifically, we should prefer to navigate directly to the fixed time url rather than clicking in the UI

Reply:
so we aren't testing the TC directly, no, I grabbed this code from the test above, basically I want to test it in realtime so that the table length is fluctuating with the incoming and outgoing telemetry, to make sure the auto scroll is working in that dynamic situation. Is there a better way?